### ISSUE TITLE
trying to update from 4.3.1 to 4.3.3 gives error

### ISSUE THREAD

**Issue by pvtex** [joBr99/nspanel-lovelace-ui#1061]
### PROBLEM DESCRIPTION
trying to update from 4.3.1 to 4.3.3 gives this error:
`Logger: custom_components.hacs
Source: custom_components/hacs/repositories/base.py:991
Integration: HACS ([documentation](https://hacs.xyz/docs/configuration/start), [issues](https://github.com/hacs/integration/issues))
First occurred: 13:42:09 (132 occurrences)
Last logged: 13:42:09

<Appdaemon joBr99/nspanel-lovelace-ui> Download was not completed [[Errno 17] File exists: '/config/appdaemon']` 

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Provide your apps.yaml (or your config):
```
  apps.yaml here:
`nspanel-1:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/nspanel/RESULT"
    panelSendTopic: "cmnd/nspanel/CustomSend"
    model: eu
    sleepTimeout: 3600
    sleepBrightness:
      - time: "7:00:00"
        value: 10
      - time: "23:00:00"
        value: 5
    locale: "de_DE"
    screensaver:
      type: screensaver2
      entities:
        - entity: weather.home
        - entity: sensor.home_hours_of_sun_0d
          name: Hours of Sun
          icon: mdi:weather-sunny
        - entity: sensor.home_uv_index
        - entity: sensor.home_wind
        - entity: sensor.mb_grid_l3_active_power
          name: L3
          icon: mdi:transmission-tower
        - entity: sensor.mb_solar_active_power_inv
          name: Solar
          icon: mdi:solar-panel
        - entity: sensor.victron_system_battery_state
          name: Storage
          icon: mdi:fuel-cell
        - entity: sensor.victron_system_battery_power
          name: Power
          icon: mdi:fuel-cell
        - entity: sensor.victron_battery_soc_238
          name: Battery
        - entity: sensor.battery_timetogo
          name: Time2Go
          icon: mdi:clock-end
    hiddenCards:
      - type: cardMedia
        key: kinocontrol
        entity: media_player.marantz_sr7010
        title: AVR Control
    cards:
      - type: cardEntities
        entities:
          - entity: select.kino_activities
          - entity: light.zengge_33_6b8f9c_2462ab6b8f9c
          - entity: navigate.kinocontrol
        title: Kino
      - type: cardPower
        title: Solar
        entities:
          - entity: sensor.victron_system_battery_state
            icon: mdi:fuel-cell
          - entity: delete
          - entity: sensor.mb_grid_l3_active_power
            name: Grid L3
            icon: mdi:transmission-tower
            speed: -100
          - entity: sensor.mb_soyo_active_power
            name: Inverter
            icon: mdi:battery-arrow-up
            speed: >-
              {% set entity_power = states('sensor.mb_soyo_active_power') | float(0) | round(3)%}
              {% set total_power = (states('sensor.mb_grid_l3_active_power') | float(0)) + (states('sensor.mb_solar_active_power_inv') | float(0)) | round(3) %}
              {% set entity_usage = (entity_power / total_power * 100) | float(0) %}
              {{ (entity_usage | round()) * -1 }}
          - entity: sensor.mb_battery_active_power
            name: Charger
            icon: mdi:battery-charging
            speed: >-
              {% set entity_power = states('sensor.mb_battery_active_power') | float(0) | round(3)%}
              {% set total_power = (states('sensor.mb_grid_l3_active_power') | float(0)) + (states('sensor.mb_solar_active_power_inv') | float(0)) | round(3) %}
              {% set entity_usage = (entity_power / total_power * 100) | float(0) %}
              {{ (entity_usage | round()) }}
          - entity: sensor.mb_solar_active_power_inv
            name: Solar
            icon: mdi:solar-panel
            speed: >-
              {% set entity_power = states('sensor.mb_solar_active_power_inv') | float(0) | round(3)%}
              {% set total_power = (states('sensor.mb_grid_l3_active_power') | float(0)) + (states('sensor.mb_solar_active_power_inv') | float(0)) | round(3) %}
              {% set entity_usage = (entity_power / total_power * 100) | float(0) %}
              {{ (entity_usage | round()) }}
          - entity: sensor.mb_herdl3_kuehlschrank_active_power
            name: Herd L3, Kuehlschrank
            speed: >-
              {% set entity_power = states('sensor.mb_herdl3_kuehlschrank_active_power') | float(0) | round(3)%}
              {% set total_power = (states('sensor.mb_grid_l3_active_power') | float(0)) + (states('sensor.mb_solar_active_power_inv') | float(0)) | round(3) %}
              {% set entity_usage = (entity_power / total_power * 100) | float(0) %}
              {{ (entity_usage | round()) * -1 }}
            icon: mdi:fridge
          - entity: sensor.mb_keller_flur_gaestewc_active_power
            name: Keller, Flur, GaesteWC
            speed: >-
              {% set entity_power = states('sensor.mb_keller_flur_gaestewc_active_power') | float(0) | round(3)%}
              {% set total_power = (states('sensor.mb_grid_l3_active_power') | float(0)) + (states('sensor.mb_solar_active_power_inv') | float(0)) | round(3) %}
              {% set entity_usage = (entity_power / total_power * 100) | float(0) %}
              {{ (entity_usage | round()) * -1 }}
            icon: mdi:home-floor-negative-1
      - type: cardGrid
        title: Solar Control
        entities:
          - entity: delete
          - entity: switch.tasmota
            name: Charger
            icon: mdi:battery-charging
          - service: automation.toggle
            entity: automation.ladegeraet_ausschalten
            icon: mdi:battery-charging
            name: Charger Auto
          - entity: delete
          - entity: switch.soyodisplay_soyosource_gtn_display_emergency_power_off
            name: Inv Em-Off
            icon: mdi:battery-charging-outline
`

```
- [ ] Go to Settings>Add-ons>AppDaemon>Log (or the output of your TS script) and then, provide the output of the log during your issue/bug occurs:
```
  Log output here:
`Logger: custom_components.hacs
Source: custom_components/hacs/repositories/base.py:991
Integration: HACS ([documentation](https://hacs.xyz/docs/configuration/start), [issues](https://github.com/hacs/integration/issues))
First occurred: 13:42:09 (132 occurrences)
Last logged: 13:42:09

<Appdaemon joBr99/nspanel-lovelace-ui> Download was not completed [[Errno 17] File exists: '/config/appdaemon']`

```

### TO REPRODUCE
_Steps to reproduce the behavior:_
update from 4.3.1 to 4.3.3

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
should update

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
_Please note here in case you are using ioBroker_

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_

EU

**Comment by joBr99** [comment]
Just replace the content of `apps/nspanel-lovelace-ui/luibackend/pages.py` manually with this:

https://raw.githubusercontent.com/joBr99/nspanel-lovelace-ui/v4.3.3/apps/nspanel-lovelace-ui/luibackend/pages.py

This is going to be the last release based on AppDaemon anyways.

**Comment by pvtex** [comment]
thank you just replaced the file.
but update give the same error.
I will wait for next release without appdaemon

**Comment by joBr99** [comment]
ignore the update message in hacs; you did the update manually by replacing that file

**Comment by pvtex** [comment]
thought there where more changes from 4.3.1 to 4.3.3

**Comment by joBr99** [comment]
no, just a fix to be compatible with HA 2023.11
